server/docker/docker-compose.yml

```diff
@@ -9,8 +9,6 @@ services:
       POSTGRES_USER: ${POSTGRES_USER}
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_DB: ${POSTGRES_DB}
-    volumes:
-      - ${POSTGRES_DATA_PATH}:/var/lib/postgresql/data
```

Comment:
I'm going to open a separate issue/PR for this. It was breaking dev build of server

Reply:
#2689